SpiffWorkflow/bpmn/parser/node_parser.py

```diff
@@ -0,0 +1,46 @@
+from .util import xpath_eval, first
+
+CAMUNDA_MODEL_NS = 'http://camunda.org/schema/1.0/bpmn'
+
+class NodeParser:
```

Comment:
My vision for the parser is to actually not have a task parsers as they're currently implemented but node parsers.  It would be reasonable for users to want to write other parsers besides task parsers, so it would be better if our parser wasn't based on tasks, but XML nodes.  Our parser ought to have the ability to call a custom parser for any XML element.  This change doesn't get us there 100% but it's a step in that direction.